firmware/src/gnss.cpp

```diff
@@ -51,6 +51,7 @@ os::config::Param<unsigned> param_gnss_aux_prio("uavcan.prio-aux",
                                                 uavcan::TransferPriority::NumericallyMin,
                                                 uavcan::TransferPriority::NumericallyMax);
 
+os::config::Param<unsigned> param_gnss_nav_filter_model("gnss.nav_filter_model", 8, 0, 9);
```

Comment:
Sorry that it is not immediately clear, but maximum parameter name length must not exceed 16 characters. This limitation is imposed by UAVCAN-MAVLink parameter bridge.